packages/front-end/components/License/DownloadLicenseUsageButton.tsx

```diff
@@ -32,8 +33,12 @@ const DownloadLicenseUsageButton: FC = () => {
             {
               license: license,
               licenseMetaData: res.licenseMetaData,
-              userLicenseCodes: res.userLicenseCodes,
-              seatsUsed: res.userLicenseCodes.length,
+              userEmailCodes: res.userEmailCodes,
+              inviteEmailCodes: res.inviteEmailCodes,
+              activeSeatsUsed: res.userEmailCodes.length,
+              seatsUsed: Array.from(
+                new Set(res.userEmailCodes.concat(res.inviteEmailCodes))
+              ).length,
```

Comment:
```suggestion
              seatsUsed: 
                new Set(res.userEmailCodes.concat(res.inviteEmailCodes).size,
```
You should be able to do set.size to get the length. No need to convert to an array